File: laravel/app/Models/PageComment.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class PageComment extends Model
{
    protected $table = 'zetawiki.page_comments';

    public $timestamps = false;

    protected $fillable = [
        'page_id',
        'message',
        'user_id',
        'user_name',
        'created',
    ];

    protected $casts = [
        'id' => 'int',
        'page_id' => 'int',
        'user_id' => 'int',
        'created' => 'datetime',
    ];

    public function page()
    {
        return $this->belongsTo(Page::class, 'page_id', 'page_id');
    }
}

File: goapp/models/page_comment.go
package models

import "time"

type PageComment struct {
	ID       int       `json:"id" gorm:"column:id"`
	PageID   int       `json:"page_id" gorm:"column:page_id"`
	UserID   int       `json:"user_id" gorm:"column:user_id"`
	UserName string    `json:"user_name" gorm:"column:user_name"`
	Created  time.Time `json:"created" gorm:"column:created"`
	Message  string    `json:"message" gorm:"column:message"`
}

func (PageComment) TableName() string {
	return "zetawiki.page_comments"
}
